### PR TITLE
semantic map based tracking

### PR DESCRIPTION
**Pull request by weidezhang**
1. use prediction trajectory to help semantic map tracking
2. currently experimental mode

### CODE REVIEW

File: modules/perception/lidar/lib/tracker/semantic_map/evaluator_manager.h [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (Capri2014) [inline modules/perception/lidar/lib/tracker/semantic_map/evaluator_manager.h:2]: 2020

**Comment by jinghaomiao** [comment]
@weidezhang seems like CLA check failed.

File: modules/perception/lidar/lib/tracker/semantic_map/evaluator_manager.cc [hunk 2]
```diff
@@ -0,0 +1,133 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
```
Comment (jinghaomiao) [inline modules/perception/lidar/lib/tracker/semantic_map/evaluator_manager.cc:2]: 2020

**Comment by weidezhang** [comment]
> @weidezhang seems like CLA check failed.

**Pull request closed without merging**